### PR TITLE
Update PKGBUILD

### PR DESCRIPTION
**Pull request by ljmf00**
This add `git` to makedepends, as it use a git source.

Please make sure you build every package on a clean chroot
@bionade24

### CODE REVIEW

**Comment by bionade24** [comment]
I never built this package as I not that far with updating the PKGBUILDs, that's why this is missing. I would even think this should be a `-git` package. When you'll update the .SRCINFO, too, I'll merge the commit.

**Comment by acxz** [comment]
@ljmf00, mentioning this here.

Thanks submitting your PRs it is much appreciated however, in all your PRs that I have looked at you have not updated the .SRCINFO file, please go ahead and do so along with the PKGBUILD file. I suggest you read the following page: https://wiki.archlinux.org/index.php/.SRCINFO if you havent already and understand the AUR packaging pipeline a bit better.

Another thing to note is it the conversation involved in the following PR: https://github.com/ros-melodic-arch/ros-melodic-laser-filters/pull/1

Basically it would be much appreciated when submitting PRs to the packages you also make sure they align with the official package source instead of `ros-gbp`.

Again thank you for your efforts, we would love the help for maintaining these ros packages, but let us try to meet halfway and get these fixed properly!
